views/update/overseas-entity-query.html

```diff
@@ -1,39 +1,53 @@
 {% extends "layout.html" %}
 
-{% set title = "What is the OE number? (NOT LIVE)" %}
+{% set title = "What is the Overseas Entity ID?" %}
 
 {% block pageTitle %}
-    {% include "includes/page-title.html" %}
+  {% include "includes/page-title.html" %}
 {% endblock %}
 
-{% block beforeContent %}
-  {% include "includes/phase_banner.html" %}
-  {% include "includes/no-back-link.html" %}
+{% block backLink %}
+  {% include "includes/back-link.html" %}
 {% endblock %}
 
 {% block content %}
-
-<div class="govuk-grid-row">
+  <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">{{ title }}</h1>
+      {% include "includes/list/errors.html" %}
+      <p class="govuk-caption-m govuk-!-margin-bottom-4">The Overseas Entity ID will be in the format OE123456.</p>
```

Comment:
can't this text added as hint in govukInput

Reply:
good shout!

Reply:
I tried this out and it seems like it will get included in the error if I do that
<img width="748" alt="image" src="https://user-images.githubusercontent.com/117645123/220082818-d2fea283-4560-4f48-94b8-54944fc4d281.png">

Reply:
Going with the original code after discussing with the BA

Reply:
@jkayyala after further discussion with BA, I will be going with your suggestion and adding it as hint text